src/reconstruction/modelGeneration/massBalance/computeMetFormulae.m

```diff
@@ -916,4 +994,22 @@
     k = floor(index / N);
 end
 s = [charSet(index + 1) s];
+end
+
+function metDup = findDuplicatedMets(model)
```

Comment:
May you put this internal function as a separate one, including documentation?

Reply:
@laurentheirendt, I used `checkCobraModelUnique` instead. Pretty much the same, just added a flag stating whether the model is unique.

Reply:
Could we add then that functionality to `checkCobraModelUnique` instead of redefining an internal function?

Reply:
Yes, it had already been removed since I changed to using `checkCobraModelUnique`.